test/integration/dialects/postgres/query-interface.test.js

```diff
@@ -8,13 +8,151 @@ var chai = require('chai')
   , DataTypes = require(__dirname + '/../../../../lib/data-types')
   , _ = require('lodash');
 
+
 if (dialect.match(/^postgres/)) {
   describe('[POSTGRES Specific] QueryInterface', function () {
     beforeEach(function () {
       this.sequelize.options.quoteIdenifiers = true;
       this.queryInterface = this.sequelize.getQueryInterface();
     });
 
+    describe('createFunction', function () {
+
+      beforeEach( function () {
```

Comment:
`beforeEach` and `afterEach` both are doing the cleanup work and duplicating it. In each test we are creating a function that we should remove before next test. You can just get rid of `beforeEach` , because `afterEach` is already doing the cleanup work